community_images/fluent-bit/ironbank/config/fluent-bit.config

```diff
@@ -0,0 +1,69 @@
+[SERVICE]
+    # Flush
+   
+    flush        1
+
+    # Daemon
+    
+    daemon       Off
```

Comment:
Are we exercising daemon on somewhere?

Reply:
yes, but that is for running this fluent-bit.config file through command

Reply:
Simple solution, create two config files with only daemon on/off difference -> use one in docker-compose and the other in k8s runtime.

Reply:
Done